zerver/openapi/zulip.yaml

```diff
@@ -1869,6 +1869,49 @@ paths:
                                     [["Denmark", "topic", 1594825442]],
                                   "id": 0,
                                 }
+                            - type: object
+                              description: |
+                                Event sent to a user's clients when that user's set of
+                                configured user-topic relationships have changed.
+                              properties:
+                                id:
+                                  $ref: "#/components/schemas/EventIdSchema"
+                                type:
+                                  allOf:
+                                    - $ref: "#/components/schemas/EventTypeSchema"
+                                    - enum:
+                                        - user_topic
+                                stream_id:
+                                  type: integer
+                                  description: |
+                                    The ID of the stream to which the topic belongs.
+                                topic_name:
+                                  type: string
+                                  description: |
+                                    The name of the topic.
+                                last_updated:
+                                  type: integer
+                                  description: |
+                                    An integer UNIX timestamp representing when the user-topic
+                                    relationship was changed.
+                                visibility_policy:
+                                  type: integer
+                                  description: |
+                                    The type of visibility policy followed by the user-topic pair.
+
+                                    - 0 - "None", used in events to indicate this topic no longer
+                                      has any user_topic configuration.
```

Comment:
Should this be a dash (-) and not an underscore for consistency? Also, is there no configuration or is it now following the user-stream configuration (e.g. if the stream is muted, the topic would be muted)?

Reply:
Right. This should have been "default user-topic configuration".